codespell_lib/data/dictionary.txt

```diff
@@ -17826,6 +17831,7 @@ varients->variants
 varierty->variety
 variey->variety
 varified->verified
+varify->verify
```

Comment:
Seems this is also a real word:
https://www.dictionary.com/browse/varify

Reply:
Mhhh, neither Mozilla/Firefox nor LibreOffice spelling corrections are accepting this as a correct word.

There seems to be also already a "varifying->verifying" in the dictionary.txt one line below. 

On the other hand https://en.wiktionary.org/wiki/varify is available as well, so any opinions to remove varifying->verifying instead?

Reply:
Probably also `varified->verified`.

I accept it's a rarely used word, but it is a valid word. See also discussion in #712 .

Reply:
I have removed the ``varify->verify`` from this PR again and created a separate PR #1127 for the other variants.